sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/DescribeTableSuite.scala

```diff
@@ -237,6 +237,34 @@ trait DescribeTableSuiteBase extends command.DescribeTableSuiteBase
     }
   }
 
+  test("DESCRIBE table_name column_name should show default value") {
```

Comment:
This test is put in `v1.DescribeTableSuiteBase`. Does v2 tables have the same issue?